src/app/map/page.tsx

```
@@ -1,10 +1,12 @@
 "use client";
 
+import KakaoMap from "@/components/KakaoMap";
 import Link from "next/link";
 
 export default function Map() {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)